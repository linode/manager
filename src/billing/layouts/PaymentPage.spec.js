describe('billing/PaymentPage', () => {
  it.skip('should render without error', () => {
  });
});
