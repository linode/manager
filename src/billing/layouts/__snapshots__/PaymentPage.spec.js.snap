// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`billing/PaymentPage should render without error 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <PaymentPage
    dispatch={[Function]}
    payment={Object {}}
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <SideEffect(ChainedDocumentTitle)
          title="Payment #undefined"
        />,
        <UNDEFINED
          crumbs={
            Array [
              Object {
                "label": "History",
                "to": "/billing/history",
              },
              Object {
                "label": "Payment #undefined",
              },
            ]
          }
        />,
        <section>
          <Card
            className=""
            header={
              <CardHeader
                title="Payment #undefined"
              />
            }
          >
            <Table
              className=""
              columns={
                Array [
                  Object {
                    "dataKey": "id",
                    "formatFn": [Function],
                    "label": "Description",
                  },
                  Object {
                    "dataKey": "date",
                    "formatFn": [Function],
                    "headerClassName": "DateColumn",
                    "label": "From",
                  },
                  Object {
                    "className": "text-right",
                    "dataKey": "usd",
                    "formatFn": [Function],
                    "headerClassName": "IntegerColumn text-right",
                    "label": "Amount",
                  },
                ]
              }
              data={
                Array [
                  Object {},
                ]
              }
              disableHeader={false}
              noDataMessage="No Payments found."
            />
            <div
              className="row"
            >
              <div
                className="col-sm-12 text-right"
              >
                <strong>
                  Payment Total: 
                  <Currency
                    value={NaN}
                  />
                </strong>
              </div>
            </div>
          </Card>
        </section>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "title": "Payment #undefined",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "crumbs": Array [
            Object {
              "label": "History",
              "to": "/billing/history",
            },
            Object {
              "label": "Payment #undefined",
            },
          ],
        },
        "ref": null,
        "rendered": null,
        "type": undefined,
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <Card
            className=""
            header={
              <CardHeader
                title="Payment #undefined"
              />
            }
          >
            <Table
              className=""
              columns={
                Array [
                  Object {
                    "dataKey": "id",
                    "formatFn": [Function],
                    "label": "Description",
                  },
                  Object {
                    "dataKey": "date",
                    "formatFn": [Function],
                    "headerClassName": "DateColumn",
                    "label": "From",
                  },
                  Object {
                    "className": "text-right",
                    "dataKey": "usd",
                    "formatFn": [Function],
                    "headerClassName": "IntegerColumn text-right",
                    "label": "Amount",
                  },
                ]
              }
              data={
                Array [
                  Object {},
                ]
              }
              disableHeader={false}
              noDataMessage="No Payments found."
            />
            <div
              className="row"
            >
              <div
                className="col-sm-12 text-right"
              >
                <strong>
                  Payment Total: 
                  <Currency
                    value={NaN}
                  />
                </strong>
              </div>
            </div>
          </Card>,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "children": Array [
              <Table
                className=""
                columns={
                  Array [
                    Object {
                      "dataKey": "id",
                      "formatFn": [Function],
                      "label": "Description",
                    },
                    Object {
                      "dataKey": "date",
                      "formatFn": [Function],
                      "headerClassName": "DateColumn",
                      "label": "From",
                    },
                    Object {
                      "className": "text-right",
                      "dataKey": "usd",
                      "formatFn": [Function],
                      "headerClassName": "IntegerColumn text-right",
                      "label": "Amount",
                    },
                  ]
                }
                data={
                  Array [
                    Object {},
                  ]
                }
                disableHeader={false}
                noDataMessage="No Payments found."
              />,
              <div
                className="row"
              >
                <div
                  className="col-sm-12 text-right"
                >
                  <strong>
                    Payment Total: 
                    <Currency
                      value={NaN}
                    />
                  </strong>
                </div>
              </div>,
            ],
            "className": "",
            "header": <CardHeader
              title="Payment #undefined"
            />,
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "className": "",
                "columns": Array [
                  Object {
                    "dataKey": "id",
                    "formatFn": [Function],
                    "label": "Description",
                  },
                  Object {
                    "dataKey": "date",
                    "formatFn": [Function],
                    "headerClassName": "DateColumn",
                    "label": "From",
                  },
                  Object {
                    "className": "text-right",
                    "dataKey": "usd",
                    "formatFn": [Function],
                    "headerClassName": "IntegerColumn text-right",
                    "label": "Amount",
                  },
                ],
                "data": Array [
                  Object {},
                ],
                "disableHeader": false,
                "noDataMessage": "No Payments found.",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <div
                  className="col-sm-12 text-right"
                >
                  <strong>
                    Payment Total: 
                    <Currency
                      value={NaN}
                    />
                  </strong>
                </div>,
                "className": "row",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <strong>
                    Payment Total: 
                    <Currency
                      value={NaN}
                    />
                  </strong>,
                  "className": "col-sm-12 text-right",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": Array [
                      "Payment Total: ",
                      <Currency
                        value={NaN}
                      />,
                    ],
                  },
                  "ref": null,
                  "rendered": Array [
                    "Payment Total: ",
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "function",
                      "props": Object {
                        "value": NaN,
                      },
                      "ref": null,
                      "rendered": null,
                      "type": [Function],
                    },
                  ],
                  "type": "strong",
                },
                "type": "div",
              },
              "type": "div",
            },
          ],
          "type": [Function],
        },
        "type": "section",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <SideEffect(ChainedDocumentTitle)
            title="Payment #undefined"
          />,
          <UNDEFINED
            crumbs={
              Array [
                Object {
                  "label": "History",
                  "to": "/billing/history",
                },
                Object {
                  "label": "Payment #undefined",
                },
              ]
            }
          />,
          <section>
            <Card
              className=""
              header={
                <CardHeader
                  title="Payment #undefined"
                />
              }
            >
              <Table
                className=""
                columns={
                  Array [
                    Object {
                      "dataKey": "id",
                      "formatFn": [Function],
                      "label": "Description",
                    },
                    Object {
                      "dataKey": "date",
                      "formatFn": [Function],
                      "headerClassName": "DateColumn",
                      "label": "From",
                    },
                    Object {
                      "className": "text-right",
                      "dataKey": "usd",
                      "formatFn": [Function],
                      "headerClassName": "IntegerColumn text-right",
                      "label": "Amount",
                    },
                  ]
                }
                data={
                  Array [
                    Object {},
                  ]
                }
                disableHeader={false}
                noDataMessage="No Payments found."
              />
              <div
                className="row"
              >
                <div
                  className="col-sm-12 text-right"
                >
                  <strong>
                    Payment Total: 
                    <Currency
                      value={NaN}
                    />
                  </strong>
                </div>
              </div>
            </Card>
          </section>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "title": "Payment #undefined",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "crumbs": Array [
              Object {
                "label": "History",
                "to": "/billing/history",
              },
              Object {
                "label": "Payment #undefined",
              },
            ],
          },
          "ref": null,
          "rendered": null,
          "type": undefined,
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <Card
              className=""
              header={
                <CardHeader
                  title="Payment #undefined"
                />
              }
            >
              <Table
                className=""
                columns={
                  Array [
                    Object {
                      "dataKey": "id",
                      "formatFn": [Function],
                      "label": "Description",
                    },
                    Object {
                      "dataKey": "date",
                      "formatFn": [Function],
                      "headerClassName": "DateColumn",
                      "label": "From",
                    },
                    Object {
                      "className": "text-right",
                      "dataKey": "usd",
                      "formatFn": [Function],
                      "headerClassName": "IntegerColumn text-right",
                      "label": "Amount",
                    },
                  ]
                }
                data={
                  Array [
                    Object {},
                  ]
                }
                disableHeader={false}
                noDataMessage="No Payments found."
              />
              <div
                className="row"
              >
                <div
                  className="col-sm-12 text-right"
                >
                  <strong>
                    Payment Total: 
                    <Currency
                      value={NaN}
                    />
                  </strong>
                </div>
              </div>
            </Card>,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "children": Array [
                <Table
                  className=""
                  columns={
                    Array [
                      Object {
                        "dataKey": "id",
                        "formatFn": [Function],
                        "label": "Description",
                      },
                      Object {
                        "dataKey": "date",
                        "formatFn": [Function],
                        "headerClassName": "DateColumn",
                        "label": "From",
                      },
                      Object {
                        "className": "text-right",
                        "dataKey": "usd",
                        "formatFn": [Function],
                        "headerClassName": "IntegerColumn text-right",
                        "label": "Amount",
                      },
                    ]
                  }
                  data={
                    Array [
                      Object {},
                    ]
                  }
                  disableHeader={false}
                  noDataMessage="No Payments found."
                />,
                <div
                  className="row"
                >
                  <div
                    className="col-sm-12 text-right"
                  >
                    <strong>
                      Payment Total: 
                      <Currency
                        value={NaN}
                      />
                    </strong>
                  </div>
                </div>,
              ],
              "className": "",
              "header": <CardHeader
                title="Payment #undefined"
              />,
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "className": "",
                  "columns": Array [
                    Object {
                      "dataKey": "id",
                      "formatFn": [Function],
                      "label": "Description",
                    },
                    Object {
                      "dataKey": "date",
                      "formatFn": [Function],
                      "headerClassName": "DateColumn",
                      "label": "From",
                    },
                    Object {
                      "className": "text-right",
                      "dataKey": "usd",
                      "formatFn": [Function],
                      "headerClassName": "IntegerColumn text-right",
                      "label": "Amount",
                    },
                  ],
                  "data": Array [
                    Object {},
                  ],
                  "disableHeader": false,
                  "noDataMessage": "No Payments found.",
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <div
                    className="col-sm-12 text-right"
                  >
                    <strong>
                      Payment Total: 
                      <Currency
                        value={NaN}
                      />
                    </strong>
                  </div>,
                  "className": "row",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": <strong>
                      Payment Total: 
                      <Currency
                        value={NaN}
                      />
                    </strong>,
                    "className": "col-sm-12 text-right",
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": Array [
                        "Payment Total: ",
                        <Currency
                          value={NaN}
                        />,
                      ],
                    },
                    "ref": null,
                    "rendered": Array [
                      "Payment Total: ",
                      Object {
                        "instance": null,
                        "key": undefined,
                        "nodeType": "function",
                        "props": Object {
                          "value": NaN,
                        },
                        "ref": null,
                        "rendered": null,
                        "type": [Function],
                      },
                    ],
                    "type": "strong",
                  },
                  "type": "div",
                },
                "type": "div",
              },
            ],
            "type": [Function],
          },
          "type": "section",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactFifteenAdapter {
      "options": Object {
        "supportPrevContextArgumentOfComponentDidUpdate": true,
      },
    },
  },
}
`;
